CHANGELOG.md

```diff
@@ -30,6 +30,8 @@
 * Added the `NavigationMapView.showsCongestionForAlternativeRoutes` property to show congestion levels with different colors on alternative route lines. ([#2887](https://github.com/mapbox/mapbox-navigation-ios/pull/2887))
 * Added the `NavigationMapView.navigationMapView(_:didAdd:)` and `NavigationViewController.navigationViewController(_:didAdd:)` delegate methods, which will be called whenever final destination `PointAnnotation` is added to `NavigationMapView` or `NavigationViewController` respectively. ([#2961](https://github.com/mapbox/mapbox-navigation-ios/pull/2961))
 * Added the ability to show specific route leg with color overriding. When specifying the `legIndex` in `NavigationMapView.show(_:legIndex:)`, the route line for the specific route leg would show congestion colors, while other route legs would show `NavigationMapView.routeCasingColor` by default. If not specified, all the route legs would show congestion colors. During turn-by-turn navigation, the default specified route leg is the current route leg. The route leg colors could be overridden by `NavigationMapView.routeCasingColor` and the congestion colors. ([#2833](https://github.com/mapbox/mapbox-navigation-ios/pull/2833))
+* Added the ability to customzie the zoom level to show and hide the maneuver arrow through the `NavigationMapView.zoomLevelToShowManeuverArrow`, which defaults to 13.0. When zoom level is smaller than the `zoomLevelToShowManeuverArrow`, maneuver arrow will hide. When zoom level is higher than the `zoomLevelToShowManeuverArrow`, maneuver arrow will appear on map.([#2984](https://github.com/mapbox/mapbox-navigation-ios/pull/2984))
```

Comment:
Nit: typo 
```suggestion
* Added the ability to customize the zoom level to show and hide the maneuver arrow through the `NavigationMapView.zoomLevelToShowManeuverArrow`, which defaults to 13.0. When zoom level is smaller than the `zoomLevelToShowManeuverArrow`, maneuver arrow will hide. When zoom level is higher than the `zoomLevelToShowManeuverArrow`, maneuver arrow will appear on map.([#2984](https://github.com/mapbox/mapbox-navigation-ios/pull/2984))
```

Reply:
Thanks for your suggestion! I'll fix this part.